devel/cmake/Portfile

```diff
@@ -57,7 +57,7 @@ if {${subport} eq ${name}} {
     # OSX <= 10.11 does not provide "clock_gettime"
     legacysupport.newest_darwin_requires_legacy 15
     # see https://trac.macports.org/ticket/59832
-    legacysupport.redirect_bins cmake
+    legacysupport.redirect_bins cmake ccmake cpack ctest
```

Comment:
Please add in a comment about "# see https://trac.macports.org/ticket/60885" too. Otherwise good to go IMHO. Might fix the error I'm getting on 10.4 PPC. ...

Reply:
will do.

Reply:
thx!